confetti.js

```diff
@@ -25,16 +31,46 @@ const bang = (addConfettiParticles) => {
   // (och med vilken hastighet) de flyger, hur mycket partiklarna
   // sprider ut sig från varandra samt ett gruppnamn som skickas in i
   // chooseParticleColor. Ändra på dessa värden och se vad som händer!
+  setTimeout(function(){
```

Comment:
Det som behövde göras i den här funktionen för att lösa uppgiften var att öka particleAmount. Du har även programmerat ditt eget konfettishow vid en timad sekvens av setTimeout, snygg lösning :)

Ett tips när du lägger in 3 anrop till addConfettiParticles som du har gjord, är att man kan se att det blir en del rader som repeteras för varje gång som du inte har vald ändra på. Du vill ändra particleAmount, xSpread, ySpread och fördröjningen till setTimeout.
Det man kan göra då är att skapa en funktion som man kan anropa med bara dom värden som man vill variera och sen gör den ett fördröjd anrop till setTimeout.

Den kan man till exempel göra som en funktion:
```
function addDelayedConfettiParticles(particleAmount, xSpread, ySpread, delay) {
  // kan prova implementera om du vill
}
```
Som man kan använda såhär `addDelayedConfettiParticles(600, 240, 80, 500)`.

Eller för läsbarhet så gillar några att skicka in ett objekt till funktionen såhär
```
function addDelayedConfettiParticles({ particleAmount, xSpread, ySpread, delay }) {
  // kan prova implementera om du vill
}
```
så man får namn på variabler som man skickar in, så anropet blir såhär:
`addDelayedConfettiParticles({ particleAmount: 600, xSpread: 240, ySpread: 80, delay: 500 })`

Sen kan man då i bang anropa addDelayedConfettiParticles 3 gånger med olika värden för att skapa in sekvens.

Ett annat tips är en sak som man ofta stöter på, att användaren klickar 2 eller flera gånger på en knapp. Om man provar klicka många gånger på confetti knappen så blir det segt. 
Ett sett att lösa det på är att sätta knappen till disabled när användaren klickar på den, och sedan återställa den till inte disabled när "showet" är färdigt :)

Man kan disable en knapp såhär till exempel:
`document.querySelector('#confettiButton').disabled = true;`
och sedan återställa med
`document.querySelector('#confettiButton').disabled = false;`

Ett tips för inspiration är att testa göra ett konfettiregn som repeterar för alltid när man klickar på knappen, och som stoppar igen när man klickar på knappen en gång till.
Då kan man välja använda setInterval som fungerar som setTimeout fast den kommer repetera för alltid tills man stoppar den igjen. Den returnerar ett ID som man kan spara och använda när man vill stoppa den.
`let intervalId = setInterval(addConfettiParticles( ... ), 500);`
Då kan man stoppa den igen med:
`clearInterval(intervalId)`

ett exempel på det:
```
let intervalId;
const bang = (addConfettiParticles) => {
  if (intervalId) {
    clearInterval(intervalId);
    intervalId = undefined;
  } else {
    intervalId = setInterval(addConfettiParticles({ ... }), 1000);
  }
};
```

(om man gör den här typ av lösning behöver man inte disable knappen eftersom man vill kunde klicka på den igen för att stoppa)